hlapipeline/utils/filter.py

```diff
@@ -0,0 +1,156 @@
+""" Utility to filter datasets which cannot be aligned 
+
+The function analyze_data opens an input list containing FLT or FLC FITS file 
+names in order to access the primary header data.  Based upon the values of specified 
+FITS keywords, the function determines whether or not each file within this dataset 
+can be reconciled against an astrometric catalog and, for multiple images, used to create 
+a mosaic.
+
+"""
+from astropy.io import fits
+from astropy.table import Table
+import math
+
+__all__ = ['analyze_data']
+
+#def analyze_data(inputFileList, optionalCols = None):
+def analyze_data(inputFileList, **kwargs):
+    """
+    Determine if images within the dataset can be aligned
+   
+    Parameters
+    ==========
+    inputFileList: list
+        List containing FLT or FLC filenames for all input images which comprise an associated 
+        dataset where 'associated dataset' may be a single image, multiple images, an association, 
+        or a number of associations
+
+    Returns
+    =======
+    outputTable: object
+        Astropy Table object containing data pertaining to the associated dataset, including 
+        the doProcess bool
+
+    The keyword/value pairs below define the "cannot process categories".
+    OBSTYPE: not IMAGING
```

Comment:
Adding a space before the colon for these keywords would allow Sphinx to recognize how to format the information according to their rules. (Unreasonably picky, but it works...)